xml/ha_rear.xml

```diff
@@ -445,10 +445,11 @@
   </para>
 
   <para>
-   A default &rear; configuration for reference is available at
-   <filename>/usr/share/rear/conf/default.conf</filename>. Other example
-   configurations (<filename>*example.conf</filename>) are available in the
-   <filename>example</filename> subdirectory. 
+   All &rear; configuration variables and their default values are set in
+   <filename>/usr/share/rear/conf/default.conf</filename>.
+   Some example files (<filename>*example.conf</filename>) for user configurations
+   (for example, what is set in <filename>/etc/rear/local.conf</filename>)
+   are available in the <filename>examples</filename> subdirectory.
    Find more information in the &rear; man page.
   </para>
 
```

Comment:
Your text is good (no different sense).
By the way:
Shouldn't words like 'simply' be avoided?